go/stellar/remote/remote.go

```diff
@@ -168,6 +168,7 @@ func PostBundleRestricted(ctx context.Context, g *libkb.GlobalContext, bundle *s
 	section["visible_parent"] = boxed.VisParentB64
 	section["version_parent"] = boxed.FormatVersionParent
 	section["account_bundles"] = boxed.AcctBundles
+	section["miniversion_parent"] = 2
```

Comment:
miniversion can go away now cause a v2 bundle can't have the miniversion 1 problem

Reply:
@mlsteele it looks like this is here because the server is sharing parsing and validation code between the two different bundle formats. it's actually only validated as being `2` if it's set. so removing this line would be fine. any idea why patrick would have added it 11 days ago though?

Reply:
executive decision. miles and i have decided to remove this.